lib/contracts/solvernet/config.go

```diff
@@ -1,19 +1,19 @@
 package solvernet
 
 import (
-	"github.com/omni-network/omni/lib/errors"
 	"github.com/omni-network/omni/lib/evmchain"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
 const (
-	None      uint8 = iota // 0
-	OmniCore  uint8 = 1    // 1
-	Hyperlane uint8 = 2    // 2
+	ProviderNone uint8 = iota // No provider
+	ProviderCore uint8 = 1    // Omni Core
+	ProviderHL   uint8 = 2    // Hyperlane
```

Comment:
Generally good to prefix exported constants with their "type" / "namespace"

examples: 
- `evmchain.IDEthereum`
- `cctp.MsgStatusSubmitted`
- `solvertypes.RejectNone`